src/containers/views/Configuration.js

```diff
@@ -73,6 +73,7 @@ export class Configuration extends Component {
     return (
       <HotKeys handlers={keyboardHandlers} className="single">
         {errors && errors.length > 0 && <Errors errors={errors} />}
+        {setTitleTag('Configuration')}
```

Comment:
consistency FTW ?  :grin:
Didn't want `Jekyll Admin` to crop up as the tab title for no reason..